azkaban-common/src/test/java/azkaban/executor/ExecutorManagerTest.java

```diff
@@ -118,15 +122,13 @@ public void testLocalExecutorScenario() throws ExecutorManagerException {
    */
   @Test
   public void testMultipleExecutorScenario() throws ExecutorManagerException {
-    final Props props = new Props();
-    props.put(ExecutorManager.AZKABAN_USE_MULTIPLE_EXECUTORS, "true");
+    this.props.put(ExecutorManager.AZKABAN_USE_MULTIPLE_EXECUTORS, "true");
```

Comment:
In future refactor, I think we can have a setup method to have all properties. When some @Test doesn't need, we can do props.delete().